app/views/shared/messages/_chat_box.html.erb

```diff
@@ -1,41 +1,94 @@
-<h1>Messages</h1>
-<h1><%= @group.name %></h1>
-<%= render 'shared/messages/form', group: @group, message: @message %>
-<h2>Messages</h2>
-<div class="bottom" id="comment_section">
-  <% @group.messages.order(created_at: :desc).each do |message| %>
-    <% if !message.show %>
-      <div class="card" id="comment-<%= message.id %>">
-        <p>Message deleted</p>
-      </div>
-    <% else %>
-      <div class="card" id="comment-<%= message.id %>">
-        <p>Message: <%= message.content %></p>
-        <p>Email: <%= message.user.email %></p>
-        <%= link_to "Delete message", message_path(message), method: :delete, data: { confirm: "Are you sure?" }, remote: true %>
-      </div>
+<div class="container mt-5 mb-5" style="background-color: white;">
```

Comment:
Please try to check that when you are typing a message, and someone sends one, your message does not disappear. Advance.